### PR TITLE
Convert to es6

### PR DESCRIPTION
**Pull request by aaddaamm**

### CODE REVIEW

File: client/src/components/helios/index.js [hunk 1]
```diff
@@ -61,7 +61,9 @@ module.exports = React.createClass({
         <div className="location-watermark">{this.state.watermark}</div>
         <DateStamp />
         <div className="time">
-          <div className="primary-clock"><Clock width="300" height="300" /></div>
+          <div className="primary-clock">
```
Comment (begizi) [inline client/src/components/helios/index.js]: Convert to es6??

**Comment by begizi** [comment]
Needs rebase

**Comment by aaddaamm** [comment]
@begizi updated. @THEKaminsky I had to rebase your changes in (adding moment.js, adding alert to forecastDay). Can you take a look and make sure i didn't miss anything?

**Comment by THEKaminsky** [comment]
It looks like you hit all the new stuff 👍  LGTM

**Pull request merged**